src/tests/playwright/pages/offline-sine.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script defer type="module">
+      import { test, evaluate, assert } from './util/audit.js'
+      const sampleRate = 44100;
+      const length = 1;
+      const numChannels = 1;
+      const freq = 441;
+
+      test(new Promise((resolve) => {
+        const offlineContext = new OfflineAudioContext(numChannels, length * sampleRate, sampleRate);
+        const osc = offlineContext.createOscillator();
+        osc.type = 'sine';
+        osc.frequency.value = freq;
+        osc.connect(offlineContext.destination);
+
+        osc.start();
+        offlineContext.startRendering().then((buffer) => {
+          resolve({buffer: buffer.getChannelData(0)});
+        });
+      }));
+
+      evaluate(async () => {
+        const bufferData = (await window.test).buffer;
+        
+        assert(bufferData.length === sampleRate * length * numChannels, 'buffer length no match');
+        assert(bufferData[0] === 0, 'not sine wave');
+        assert(bufferData[1] !== 0, 'not sine wave');
+        assert(bufferData[sampleRate / freq] === 0, 'sine wave period wrong');
+
+        // returns true if all previous asserts passed
```

Comment:
Interesting - is this the convention of Playwright?

Reply:
I think I added a question to a wrong location. I was asking about L33 `assert()`.

Why do we need to return the value of an empty `assert()`?